src/common/perf_counters.h

```diff
@@ -352,6 +352,24 @@ class PerfCountersCollection
 };
 
 
+class PerfGuard {
+  const ceph::coarse_real_clock::time_point start;
+  PerfCounters* const counters;
+  const int event;
+
+public:
+  PerfGuard(PerfCounters* const counters,
+            const int event)
+  : start(ceph::coarse_real_clock::now()),
+    counters(counters),
+    event(event) {
+  }
+
+  ~PerfGuard() {
+    counters->tinc(event, ceph::coarse_real_clock::now() - start);
```

Comment:
I don't think coarse precision is enough for performance track, it's only milliseconds precision.

Reply:
Yeah, `ceph_clock_now()` uses `CLOCK_REALTIME`. @tchaikov, we you be fine with `ceph::real_clock`?

Reply:
@rzarzynski fine with me !

Reply:
@yuyuyu101, @tchaikov: just switched to `ceph::real_clock`.